gitbook-docs/keys.md

```diff
@@ -901,31 +901,32 @@ The angle the wind needs to shift to raise an alarm
 
 
 ## environment.wind.directionMagnetic
-The wind direction relative to magnetic north
+Wind direction based on speed through water (STW), relative to magnetic north
+
 
 **Units:**rad
 
 
 ## environment.wind.directionTrue
-The wind direction relative to true north
+True Wind Direction (TWD) based on speed through water (STW), relative to true north
 
 **Units:**rad
 
 
 ## environment.wind.speedApparent
-Apparent wind speed
+Apparent Wind Speed (AWS)
 
 **Units:**m/s
 
 
 ## environment.wind.speedOverGround
-Wind speed over ground (as calculated from speedApparent and vessel's speed over ground)
+Ground Wind Speed (GWS), calculated from apparent wind speed (AWS) and speed over ground (SOG)
```

Comment:
speedApparent in the original is used specifically to refer to the Signal K path, I don't want to drop that. Also I'd like to keep the word vessel.

Reply:
Ok, will revert to "speedApparent (AWS)" and "vessel's speed over ground"